src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/MetaGenomicsProgramGroup.java

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+public class MetaGenomicsProgramGroup  implements CommandLineProgramGroup {
```

Comment:
Would PathogenDiscoveryProgramGroup be clearer?

Reply:
Metagenomics, with lower-case `g` please.